extensions/podman/packages/extension/src/extension.ts

```diff
@@ -2203,17 +2203,29 @@ export async function createMachine(
     telemetryRecords.diskSize = params['podman.factory.machine.diskSize'];
   }
 
-  // image-path
-  if (params['podman.factory.machine.image-path'] && typeof params['podman.factory.machine.image-path'] === 'string') {
-    parameters.push('--image-path');
-    parameters.push(params['podman.factory.machine.image-path']);
+  const installedPodman = await getPodmanInstallation();
+  const version = installedPodman?.version;
+  const isPodmanV5OrLater = version?.startsWith('5.') ?? false;
```

Comment:
This does not scale to version 6 for example? let's use [semver](https://www.npmjs.com/package/semver)